Documentation/cmdref/cilium-agent.md

```diff
@@ -74,6 +74,7 @@ cilium-agent [flags]
       --enable-bpf-clock-probe                               Enable BPF clock source probing for more efficient tick retrieval
       --enable-bpf-masquerade                                Masquerade packets from endpoints leaving the host with BPF instead of iptables
       --enable-bpf-tproxy                                    Enable BPF-based proxy redirection, if support available
+      --enable-cluster-ip-external-access                    Enable external access to ClusterIP services
```

Comment:
What other documentation related changes should be made for this change?

Reply:
It'll be good to document this flag in the kube-proxy replacement getting started [guide](https://docs.cilium.io/en/v1.9/gettingstarted/kubeproxy-free/).

Reply:
Thanks. Added documentation.